CPAC/generate_motion_statistics/generate_motion_statistics.py

```diff
@@ -483,9 +483,7 @@ def gen_motion_parameters(movement_parameters, max_displacement,
     ]
 
     out_file = os.path.join(os.getcwd(), 'motion_parameters.txt')
-    summary_motion_params = os.path.join(os.getcwd(),
-                                         'summary_motion_params.tsv')
-    with open(out_file, 'w') as f:
```

Comment:
I think you still need https://github.com/FCP-INDI/C-PAC/blob/2cbd2d340efec3ee0aa5179e5f26067d1c68dc4e/CPAC/generate_motion_statistics/generate_motion_statistics.py#L488

Reply:
oh right! I think I deleted it by mistake. Thanks for looking through it.